### ISSUE TITLE
Update to the 17.0.5 JDK

### ISSUE THREAD

**Issue by aivinog1** [camunda/zeebe#10372]
**Is your feature request related to a problem? Please describe.**
Because there is [a bug with JDK that is breaking the `cpu-shares` parameter of the Docker environment](https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8281181) it would be cool to see updates of the JDK in the base Zeebe's Docker image.

**Describe the solution you'd like**
Just update the JDK in Zeebe's Docker Image to 17.0.5 as soon as it is available. 

**Describe alternatives you've considered**
You could switch to the 18 or even 19 JDKs but I think that you shouldn't do this in the patch versions :)

**Additional context**
The JDK bug heavily affects our Zeebe environment because we use this argument when running docker containers. As I can see right now Zeebe is using [this version of the image](https://hub.docker.com/layers/eclipse-temurin/library/eclipse-temurin/17.0.3_7-jre-focal/images/sha256-fce37e5146419a158c2199c6089fa39b92445fb2e66dc0331f8591891239ea3b?context=explore).

**Comment by aivinog1** [comment]
Also, I would like to see some patches to the 8.0.x version as well :)

**Comment by megglos** [comment]
17.0.5 is scheduled for [2022-10-18](https://www.java.com/releases/)

**Comment by npepinpe** [comment]
Seems to have been fixed in Java 17.0.4, see [the release notes](https://www.oracle.com/java/technologies/javase/17-0-4-relnotes.html#JDK-8281181).

Closed by #10810

**Comment by aivinog1** [comment]
> Seems to have been fixed in Java 17.0.4, see [the release notes](https://www.oracle.com/java/technologies/javase/17-0-4-relnotes.html#JDK-8281181).
> 
> Closed by #10810

Hey @npepinpe! But it seems to me, not in the Temurin JDK: https://foojay.io/java-17/?tab=component&version=17.0.4&quarter=072022
This is the 17.0.4 version:
<img width="1768" alt="image" src="https://user-images.githubusercontent.com/25925932/197745404-43b4c282-a8ae-454c-847c-05318749bf7f.png">
And this is the 17.0.5 version:
<img width="1768" alt="image" src="https://user-images.githubusercontent.com/25925932/197745662-9064e434-7dfe-424a-9483-55d2d872a7c8.png">

**Comment by aivinog1** [comment]
You can see links on this page: https://adoptium.net/temurin/releases/

**Comment by npepinpe** [comment]
I understood they follow the same versioning, but it seems not. Strange though that they always have the same versions, down to patches, yet they don't include exactly the same content (ignoring the commercial features obviously)

**Comment by aivinog1** [comment]
> Strange though that they always have the same versions, down to patches, yet they don't include exactly the same content (ignoring the commercial features obviously)
Yep, we thought in the same way but seems to be it is not true :) 

Also, I want to mention that it would be cool to backport this in the 8.0.x also :)

**Comment by megglos** [comment]
it's underway https://github.com/docker-library/official-images/pull/13458

**Comment by megglos** [comment]
@aivinog1 actually the stable/8.0 branch is not using digest pinning and [points to the general `17-jre-focal`](https://github.com/camunda/zeebe/blob/stable/8.0/Dockerfile#L24) tag of the temurin image. Thus the next patch release will automatically be based on the latest 17 tag, being at least the 17.0.5 one from now on.

**Comment by aivinog1** [comment]
> @aivinog1 actually the `stable/8.0` branch is not using digest pinning and [points to the general `17-jre-focal`](https://github.com/camunda/zeebe/blob/stable/8.0/Dockerfile#L24) tag of the temurin image. Thus the next patch release will automatically be based on the latest 17 tag, being at least the 17.0.5 one from now on.

Awesome, thanks :)
